Sources/ComposedUI/StackView/StackCoordinator.swift

```diff
@@ -135,6 +135,7 @@ extension StackCoordinator: SectionProviderMappingDelegate {
     public func mapping(_ mapping: SectionProviderMapping, select indexPath: IndexPath) { }
     public func mapping(_ mapping: SectionProviderMapping, deselect indexPath: IndexPath) { }
     public func mapping(_ mapping: SectionProviderMapping, move sourceIndexPath: IndexPath, to destinationIndexPath: IndexPath) { }
+    public func mappingDidInvalidateHeader(at sectionIndex: Int) {}
```

Comment:
if we're going to introduce this formally, we should include footer support at the same time.

Reply:
Done in #21